src/main/java/com/softserve/academy/spaced/repetition/controller/CourseRatingController.java

```diff
@@ -15,30 +15,39 @@
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 
+import static com.softserve.academy.spaced.repetition.controller.dto.builder.DTOBuilder.buildDtoForEntity;
 import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
 import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
 
 @RestController
+@RequestMapping("api/courses/{courseId}/ratings")
 public class CourseRatingController {
 
     @Autowired
     private CourseRatingService courseRatingService;
 
-    @GetMapping("api/course/{courseId}/rating/{id}")
-    public ResponseEntity<CourseRatingPublicDTO> getCourseRatingById(@PathVariable Long courseId, @PathVariable Long id) {
+    @GetMapping("/{id}")
+    public ResponseEntity<CourseRatingPublicDTO> getCourseRatingById(@PathVariable Long id) {
         CourseRating courseRating = courseRatingService.getCourseRatingById(id);
+        if (courseRating == null) {
```

Comment:
Maybe better to check this null inside of appropriate service method and throw exception with status `NOT_FOUND`